src/app/[locale]/page.tsx

```diff
@@ -1,5 +1,13 @@
-import styles from "./page.module.scss";
+import HomePage from "../../components/pages/Home/Home";
+import { getTopLatestExhibits } from "@/lib/exhibit";
+import { ITopLatestExhibit } from "@/interfaces/ITopLatestExhibit";
```

Comment:
what do you think about reusing src\interfaces\IExhibit.ts interface?

Reply:
I was thinking about the same approach, however, since I use lightweight model of exhibit that does not include a large set of properties in `IExhibit` I believe it would be better to define custom model instead.